### PR TITLE
Add x64 notice installing tools for debugging

### PR DESCRIPTION
**Pull request by DenisBalan**
For debugging under vscode, x64 installer is required.
After issue #693 x64 install option is available.
Comment in https://github.com/Azure/azure-functions-core-tools/issues/693#issuecomment-573294750 says that clarification in README file is needed for installing x64 bitness from choco.

This PR adds a notice in README.md file on how specify x64 bitness when installing from choco package manager.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,10 @@ To install with chocolatey:
 ```bash
 choco install azure-functions-core-tools
 ```
-
+#### Notice: To debug functions under vscode, x64 bitness is required
+```bash
+choco install azure-functions-core-tools --params "'/x64:true'"
```
Comment (ankitkumarr) [inline README.md]: `"'/x64'" should also work.
We can maybe just put that instead of `"'/x64:true'"`

**Comment by ankitkumarr** [comment]
Sorry this slipped and took me a while to get. Looks good to me, if you could just address one tiny comment and update with dev, we can merge this.

File: README.md (hunk 1 above)
Comment (DenisBalan) [reply README.md]: Done

**Comment by DenisBalan** [comment]
@ankitkumarr done, also merged dev into patch branch

**Pull request merged**